runtime/wasm/wasm_test/Makefile

```diff
@@ -0,0 +1,7 @@
+TS_FILES=$(wildcard *.ts)
+WASM_FILES=$(patsubst %.ts,%.wasm,$(TS_FILES))
+
+all: $(WASM_FILES)
+
+%.wasm: %.ts
+	@asc $< -b $@ --validate
```

Comment:
Nice! I can't read this but I also can't read the previous script which I wrote myself, and this uses make so it must be much smarter.

Reply:
Haha

Reply:
I don't like using `make` here though. It requires that you have it installed in the first place, whereas the previous script just required `bash` and was super obvious.

Also, aren't there different variants of make that may or may not work with the Makefile features we're using here? I'd rather keep it simple.

Reply:
I do find it convenient to rebuild only for changed inputs, compiling these files is fast but not instant.

Reply:
That (and a bit of reflex) was the reason why I changed it to a Makefile. I can go either way on this, but this would only require make on developer machines, and only if people change this test data, at which point you're pretty far down the rabbit hole already.

Reply:
Ok, I get that. Carry on :)

Reply:
> I forgot how crude the Makefile syntax is.
> 
> `all: $(WASM_FILES)` says "when running `make`, build all matched `.wasm` files".
> 
> `%.wasm: %.ts` says "when building a `.wasm` target, use the `.ts` file with the same name as the input.
> 
> `@asc` says "run `asc` command in a shell.
> 
> `$<` is a placeholder for the target name, e.g. `foo.wasm`.
> `$@` is a placeholder for the input name, e.g. `foo.ts`.

Almost ;) `@asc` says "run `asc` but do not print the command you're running" (without the `@` it would print)

`$<` are the dependencies of the target (everything to the right of the `:`) and `$@` is the actual target (to the right of `:`)